vm-mock/vm.js

```diff
@@ -75,26 +67,44 @@ let webModule;
  * @returns {?} ?
  */
 export default function createMockedABI(memory, createImports, instantiate, binary) {
+  const byteArrToString = (arr) => {
+    return new TextDecoder("utf-8").decode(arr);
+  };
+
+  const ptrToString = (ptr) => {
+    return byteArrToString(webModule.__getArrayBuffer(ptr));
+  };
+
+  function ptrToUint8ArrayString(ptr) {
+    return new Uint8Array(webModule.__getArrayBuffer(ptr)).toString();
+  }
+
+  const getArrayBuffer = (ptr) => {
+    return webModule.__getArrayBuffer(ptr);
+  };
+
+  const newArrayBuffer = (buffer) => {
+    return webModule.__newArrayBuffer(buffer);
+  };
```

Comment:
Why using arrow functions here instead of classic function declaration ?
I'm not sure it iproves readibility here...